data/mods/Magiclysm/Spells/animist.json

```diff
@@ -381,5 +381,32 @@
     "base_energy_cost": 6,
     "energy_source": "HP",
     "difficulty": 6
+  },
+  {
+    "id": "rupture",
+    "type": "SPELL",
+    "name": "Rupture",
+    "description": "You pour mana into an unwilling and unprepared vessel, causing cellular break-down, internal hemorrhaging, leylines to become muddled, and electronics to overcharge.  Immensely gruesome, crude, and was once illegal world-wide, not that it matters now.",
+    "effect": "attack",
+    "shape": "blast",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SOMATIC", "NO_PROJECTILE", "LOUD" ],
+    "max_level": 30,
+    "min_range": 1,
+    "max_range": 1,
+    "min_damage": 2,
+    "max_damage": 2,
+    "base_casting_time": 50,
+    "base_energy_cost": 500,
+    "spell_class": "ANIMIST",
+    "min_duration": 100,
+    "max_duration": 500,
+    "duration_increment": 15,
+    "min_dot": 10,
+    "max_dot": 100,
```

Comment:
100 damage per second for 5 seconds is way over the top. i think this whole section of the spell needs to be rethought and rebalanced.